.github/workflows/clabot-workflow.yml

```diff
@@ -0,0 +1,70 @@
+name: CLA Signature Bot
+
+on:
+  workflow_call:
+    inputs:
+      blockchain-storage-flag:
+        description: "(Optional) Store all the blockchain signatures in the ethirium blockchain as a smart contract."
+        required: false
+        type: boolean
+      blockchain-webhook-endpoint:
+        description: "(Optional) The URL to post the blockchain request to."
+        required: false
+        type: string
+        default: "https://u9afh6n36g.execute-api.eu-central-1.amazonaws.com/dev/webhook"
```

Comment:
We won't ever use this and it was an oversight I didn't remove it, please omit it from this.

Reply:
Removed from the shared workflow! It'll still exist in the action itself but has defaulted values. Another PR can be made if we want to remove it entirely from the action and code.